Modules/_testexternalinspection.c

```diff
@@ -520,10 +520,11 @@ parse_frame_object(
         return -1;
     }
 
-    if (address_of_code_object == NULL) {
+    if (address_of_code_object == 0) {
         return 0;
     }
-    return parse_code_object(pid, result, offsets, address_of_code_object, previous_frame);
+    address_of_code_object &= ~1;
```

Comment:
Wondering if we could avoid hardcoding a mystical `~1` here and use something from `pycore_stackref.h`.